client/source-map-support.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import fetch from 'unfetch'
+import {SourceMapConsumer} from 'source-map'
```

Comment:
I assume this is a kind a big library. Could you check whether we are getting rid of this in the production mode.

If not, shall we do it with some webpack magic?

Reply:
The parent file (source-map-support) is only loaded in development using the same if/else require 👍

Reply:
@timneutkens okay. Great.